### PR TITLE
More test development

### PR DESCRIPTION
**Pull request by baude**
Add a test that acutally runs more components of our code.  Just the first test now.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by Jackbaude** [comment]
You should now be ready with https://github.com/containers/netavark-dhcp-proxy/pull/18.

File: contrib/cirrus/runner.sh [hunk 1]
```diff
@@ -67,6 +67,8 @@ _run_unit_aarch64() {
 }
 
 _run_integration() {
+	dnf -y install dnsmasq jq
```
Comment (Luap99) [inline contrib/cirrus/runner.sh:70]: Is there a plan to put this into the VM images, jq should be already installed?

**Review by Luap99 — COMMENTED** [review]
LGTM

File: contrib/cirrus/runner.sh (hunk 1 above)
Comment (baude) [reply contrib/cirrus/runner.sh:70]: yup, right after this pr merges

**Pull request merged**